.github/workflows/test.yml

```diff
@@ -50,54 +42,33 @@ jobs:
         run: echo "INPUT ( libldap.so )" > /usr/lib64/libldap_r.so
       - name: Install required packages
         run: >
-          dnf install -y
-          asciidoc
-          asciidoctor
-          bandit
-          cpio
-          dnf-plugins-core
-          enchant
-          findutils
+          dnf --nodocs --setopt install_weak_deps=false -y install
+          cargo
           gcc
           git
           krb5-devel
-          krb5-server
           krb5-workstation
-          libcap-devel
+          libffi-devel
+          libpq-devel
           make
           openldap-clients
           openldap-devel
-          openssh
-          openssh-clients
           openssl-devel
-          postgresql-contrib
-          postgresql-devel
-          postgresql-server-devel
-          python-srpm-macros
+          postgresql
           python3-devel
-          python3-pycodestyle
-          python3-pylint
-          python3-pytest
-          python3-pytest-cov
-          python3.9
-          python36
+          python3-pip
+          python3-wheel
           redhat-rpm-config
-          rpm-build
-          rpmlint
-          rsync
-          tox
-          wget
-          which
       - name: Setup PostgreSQL service
         run: psql -f etc/pg/local-dev-app-user.sql postgres://postgres:test@postgres:5432/
       - name: Setup LDAP service
         run: ldapadd -c -H "ldap://testldap:1389" -x -D "cn=admin,dc=redhat,dc=com" -w "adminpassword" -f etc/openldap/local-export.ldif || true
+      - name: Install Python devel dependencies
+        run: pip3 install --no-deps -r devel-requirements.txt
```

Comment:
Running the `tox -e ci-osidb` performs the install of both requirements and devel-requirements inside the tox python environment so the only thing necessary should be the tox itself.